ballerina/http_status_code_types.bal

```diff
@@ -798,3 +798,132 @@ public type HttpVersionNotSupported record {|
+public final readonly & NotModified NOT_MODIFIED = {};
+
+# The common status code response constant of `UseProxy`.
+public final readonly & UseProxy USE_PROXY = {};
+
+# The common status code response constant of `TemporaryRedirect`.
+public final readonly & TemporaryRedirect TEMPORARY_REDIRECT = {};
+
+# The common status code response constant of `PermanentRedirect`.
+public final readonly & PermanentRedirect PERMANENT_REDIRECT = {};
+
+# The common status code response constant of `BadRequest`.
+public final readonly & BadRequest BAD_REQUEST = {};
+
+# The common status code response constant of `Unauthorized`.
+public final readonly & Unauthorized UNAUTHORIZED = {};
+
+# The common status code response constant of `PaymentRequired`.
+public final readonly & PaymentRequired PAYMENT_REQUIRED = {};
+
+# The common status code response constant of `Forbidden`.
+public final readonly & Forbidden FORBIDDEN = {};
+
+# The common status code response constant of `NotFound`.
+public final readonly & NotFound NOT_FOUND = {};
+
+# The common status code response constant of `MethodNotAllowed`.
+public final readonly & MethodNotAllowed METHOD_NOT_ALLOWED = {};
+
+# The common status code response constant of `NotAcceptable`.
+public final readonly & NotAcceptable NOT_ACCEPTABLE = {};
+
+# The common status code response constant of `ProxyAuthenticationRequired`.
+public final readonly & ProxyAuthenticationRequired PROXY_AUTHENTICATION_REQUIRED = {};
+
+# The common status code response constant of `RequestTimeout`.
+public final readonly & RequestTimeout REQUEST_TIMEOUT = {};
+
+# The common status code response constant of `Conflict`.
+public final readonly & Conflict CONFLICT = {};
+
+# The common status code response constant of `Gone`.
+public final readonly & Gone GONE = {};
+
+# The common status code response constant of `LengthRequired`.
+public final readonly & LengthRequired LENGTH_REQUIRED = {};
+
+# The common status code response constant of `PreconditionFailed`.
+public final readonly & PreconditionFailed PRECONDITION_FAILED = {};
+
+# The common status code response constant of `PayloadTooLarge`.
+public final readonly & PayloadTooLarge PAYLOAD_TOO_LARGE = {};
+
+# The common status code response constant of `UriTooLong`.
+public final readonly & UriTooLong URI_TOO_LONG = {};
+
+# The common status code response constant of `UnsupportedMediaType`.
+public final readonly & UnsupportedMediaType UNSUPPORTED_MEDIA_TYPE = {};
+
+# The common status code response constant of `RangeNotSatisfiable`.
+public final readonly & RangeNotSatisfiable RANGE_NOT_SATISFIABLE = {};
+
+# The common status code response constant of `ExpectationFailed`.
+public final readonly & ExpectationFailed EXPECTATION_FAILED = {};
+
+# The common status code response constant of `UpgradeRequired`.
+public final readonly & UpgradeRequired UPGRADE_REQUIRED = {};
+
+# The common status code response constant of `RequestHeaderFieldsTooLarge`.
```

Comment:
`TooManyRequests` is missing

Reply:
Fixed in [086eba7](https://github.com/ballerina-platform/module-ballerina-http/pull/983/commits/086eba7cbced1a0f59543b29a5ea01ebda945d53)